### PR TITLE
feat: support to waf Domains and Events #minor

### PR DESCRIPTION
**Pull request by diegoazion**

### CODE REVIEW

**Review by AndreSilveiraAzion — DISMISSED** [review]

File: internal/data_source_waf_events.go [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+				Description: "The WAF identifier.",
+				Required:    true,
+			},
+			"domains_ids": schema.ListAttribute{
+				ElementType: types.Int64Type,
+				Description: "The total number of Waf.",
+				Optional:    true,
+			},
+			"network_list_id": schema.Int64Attribute{
+				Description: "The total number of Waf.",
```
Comment (matheus-rabuske) [inline internal/data_source_waf_events.go:76]:
```suggestion
				Description: "The total number of WAF events.",
```

File: internal/data_source_waf_events.go [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+				Optional:    true,
+			},
+			"hour_range": schema.Int64Attribute{
+				Description: "The total number of Waf.",
```
Comment (matheus-rabuske) [inline internal/data_source_waf_events.go:80]:
```suggestion
				Description: "The total number of WAF events.",
```

File: internal/data_source_waf_events.go [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"country_count": schema.Int64Attribute{
+							Description: "The is the Country Count.",
+							Computed:    true,
+						},
+						"top_10_countries": schema.ListAttribute{
+							Computed:    true,
+							ElementType: types.StringType,
+							Description: "The is the Country Count.",
```
Comment (matheus-rabuske) [inline internal/data_source_waf_events.go:98]:
```suggestion
							Description: "The top 10 WAF event count countries.",
```

File: internal/data_source_waf_events.go [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+						},
+						"top_10_ips": schema.ListAttribute{
+							Computed:    true,
+							ElementType: types.StringType,
+							Description: "The is the Country Count.",
```
Comment (matheus-rabuske) [inline internal/data_source_waf_events.go:103]:
```suggestion
							Description: "The top 10 WAF event count IPs.",
```

File: internal/data_source_waf_events.go [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+						},
+						"hit_count": schema.Int64Attribute{
+							Description: "The is the Hit Count.",
+							Computed:    true,
+						},
+						"rule_id": schema.Int64Attribute{
+							Description: "Rule id off WAF.",
```
Comment (matheus-rabuske) [inline internal/data_source_waf_events.go:110]:
```suggestion
							Description: "Rule id of WAF.",
```

File: internal/data_source_waf_events.go [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+							Computed:    true,
+						},
+						"ip_count": schema.Int64Attribute{
+							Description: "The is the IP Count.",
```
Comment (matheus-rabuske) [inline internal/data_source_waf_events.go:114]:
```suggestion
							Description: "The WAF events IP count.",
```

File: internal/data_source_waf_events.go [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+							Computed:    true,
+						},
+						"match_zone": schema.StringAttribute{
+							Description: "The is the Match Zone.",
```
Comment (matheus-rabuske) [inline internal/data_source_waf_events.go:118]:
```suggestion
							Description: "The Match Zone.",
```

File: internal/data_source_waf_events.go [hunk 8]
```diff
@@ -0,0 +1,231 @@
...
+							Computed:    true,
+						},
+						"path_count": schema.Int64Attribute{
+							Description: "The is the Path Count.",
```
Comment (matheus-rabuske) [inline internal/data_source_waf_events.go:122]:
```suggestion
							Description: "The Path Count.",
```

File: internal/data_source_waf_events.go [hunk 9]
```diff
@@ -0,0 +1,231 @@
...
+							Computed:    true,
+						},
+						"matches_on": schema.StringAttribute{
+							Description: "The is the Matches On.",
```
Comment (matheus-rabuske) [inline internal/data_source_waf_events.go:126]:
```suggestion
							Description: "The Matches On.",
```

File: internal/data_source_waf_events.go [hunk 10]
```diff
@@ -0,0 +1,231 @@
...
+							Computed:    true,
+						},
+						"rule_description": schema.StringAttribute{
+							Description: "The is the Rule Description.",
```
Comment (matheus-rabuske) [inline internal/data_source_waf_events.go:130]:
```suggestion
							Description: "The Rule Description.",
```

**Review by matheus-rabuske — DISMISSED** [review]
LGTM, just some minor improvements.

**Review by matheus-rabuske — APPROVED** [review]

**Pull request merged**